test/extended/cluster/metrics/metrics.go

```diff
@@ -24,8 +24,37 @@ type BaseMetrics struct {
 
 type TestDuration struct {
 	BaseMetrics
-	StartTime    time.Time     `json:"startTime"`
-	TestDuration time.Duration `json:"testDuration"`
+	StartTime    time.Time      `json:"startTime"`
+	TestDuration time.Duration  `json:"testDuration"`
+	Steps        []StepDuration `json:"Steps"`
+}
+
+type StepDuration interface {
```

Comment:
Why do you create an alias for an empty interface?

Reply:
Removed.

Reply:
I remember why it is there now.
The slice to collect steps (in `cl.go`) needs a base type. It would not work to use struct instead of interface.